### PR TITLE
Hpc cloud bugs

### PR DESCRIPTION
**Pull request by romulogoncalves**

### CODE REVIEW

File: hadoop.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # Hadoop
 The platform uses Hadoop HDFS as a file system for Spark. For web-ui access to it use [the hadoop-name node address defined in the inventory file](https://github.com/nlesc-sherlock/emma/blob/master/ansible.md#provision) which is listening on port **50070**.
+
+To have password less ssh connections between nodes for the hadoop user, it is necessary to generate a ssh-key with the name **id_rsa**. It should be stored under **files** in the **hadoop** role.
```
Comment (sverhoeven) [inline hadoop.md]: Why do you need to have password less logins? Does HDFS not work without it?

Comment (sverhoeven) [inline hadoop.md]: Until now to configure you had using inventory and vars/ dir. Now you also need to add stuff in roles/ sub dir. Can we prevent doing this?

Comment (romulogoncalves) [inline hadoop.md]: That's what we do with key for the root login. It is stored under roles/common/files. Then we also need to change it.

Comment (romulogoncalves) [inline hadoop.md]: Maybe my text is miss leading, but for what I know:
"For setting up Hadoop on a cluster of machines, the master should be able to do a password-less ssh to start the daemons on all the slaves."

Are you saying it is anymore like that?

Comment (sverhoeven) [inline hadoop.md]: Yes, it now sounds the password less login is optional. While it is required for correct operation of HDFS. Could you add `For setting up ...` sentence to that section in hadoop.md

Comment (sverhoeven) [inline hadoop.md]: Hmm, this makes the roles harder to re-use or put them on Ansible galaxy. OK let's keep it as it is and worry about re-use later.

Comment (romulogoncalves) [inline hadoop.md]: I opened an issue so we fix this in the future:
https://github.com/nlesc-sherlock/emma/issues/57

**Comment by romulogoncalves** [comment]
The ssh keys issue is now solved with:
https://github.com/nlesc-sherlock/emma/commit/e80dea842f6256d5cb95f28c79b699dba83fd3a6
and
https://github.com/nlesc-sherlock/emma/commit/773ecb2a9ea2c1f70c9cff923d9fbda415901c3b

**Pull request merged**